sdk/formrecognizer/tests.yml

```diff
@@ -13,9 +13,9 @@ stages:
       # their claim is that the full matrix ends up stress-testing their service.
       # As such, the canary test runs should run on a reduced matrix.
       CloudConfig:
-        Prod:
-          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-          Location: $(LOCATION)
+#         Prod:
```

Comment:
@kristapratico is there a tracking issue to re-enable prod testing?

Reply:
Hi @benbp! Added the issue here: https://github.com/Azure/azure-sdk-for-python/issues/21784

Reply:
Thanks @catalinaperalta!